tests/lagrangian_detectors_increasing_rotation/plotscript.py

```diff
@@ -0,0 +1,21 @@
+from scipy import *
```

Comment:
Add a comment to say what this script does.  Doesn't appear to actually plot anything, despite name. Necessary?  Seems to overlap with analytical_traj.py.